mkosi/config.py

```diff
@@ -742,465 +751,681 @@ class MkosiConfigParser:
+            help="Make sure the image can be used as an initramfs",
         ),
         MkosiConfigSetting(
             dest="kernel_modules_include",
+            metavar="REGEX",
             section="Content",
             parse=config_make_list_parser(delimiter=","),
+            help="Only include the specified kernel modules in the image",
         ),
         MkosiConfigSetting(
             dest="kernel_modules_exclude",
+            metavar="REGEX",
             section="Content",
             parse=config_make_list_parser(delimiter=","),
+            help="Exclude the specified kernel modules from the image",
         ),
         MkosiConfigSetting(
             dest="kernel_modules_initrd",
+            metavar="BOOL",
+            nargs="?",
             section="Content",
             parse=config_parse_boolean,
             default=True,
+            help="When building a bootable image, add an extra initrd containing the kernel modules",
         ),
         MkosiConfigSetting(
             dest="kernel_modules_initrd_include",
+            metavar="REGEX",
             section="Content",
             parse=config_make_list_parser(delimiter=","),
+            help="When building a kernel modules initrd, only include the specified kernel modules",
         ),
         MkosiConfigSetting(
             dest="kernel_modules_initrd_exclude",
+            metavar="REGEX",
             section="Content",
             parse=config_make_list_parser(delimiter=","),
+            help="When building a kernel modules initrd, exclude the specified kernel modules",
         ),
         MkosiConfigSetting(
             dest="locale",
             section="Content",
             parse=config_parse_string,
+            help="Set the system locale",
         ),
         MkosiConfigSetting(
             dest="locale_messages",
+            metavar="LOCALE",
             section="Content",
             parse=config_parse_string,
+            help="Set the messages locale",
         ),
         MkosiConfigSetting(
             dest="keymap",
+            metavar="KEYMAP",
             section="Content",
             parse=config_parse_string,
+            help="Set the system keymap",
         ),
         MkosiConfigSetting(
             dest="timezone",
+            metavar="TIMEZONE",
             section="Content",
             parse=config_parse_string,
+            help="Set the system timezone",
         ),
         MkosiConfigSetting(
             dest="hostname",
+            metavar="HOSTNAME",
             section="Content",
             parse=config_parse_string,
+            help="Set the system hostname",
         ),
         MkosiConfigSetting(
             dest="root_password",
+            metavar="PASSWORD",
             section="Content",
             parse=config_parse_root_password,
             paths=("mkosi.rootpw",),
             path_read_text=True,
             path_secret=True,
+            help="Set the root's password",
```

Comment:
the combination of article and possessive marker looks weird, I think either of these alternatives would work
```suggestion
            help="Set the root password",
```
or
```suggestion
            help="Set root's password",
```
or
```suggestion
            help="Set the root account's password",
```
or
```suggestion
            help="Set the root user's password",
```

My preference would be the first one.

Reply:
Hmm, the first one is how we'd usually write it. But "set zbyszek password" looks very strange, and really "root" is just the account name here. What about "set password for root"? I think that avoids the issue nicely.